### ISSUE TITLE
doc_markdown should ignore quotes

### ISSUE THREAD

**Issue by PixelPirate** [Manishearth/rust-clippy#1277]
I have a doc comment that describes the contents of a JSON document like this:
```rust
    /// So much JSON:
    /// { "many camelCase": "wow" }
```
"camelCase" here is a false positive of doc_markdown. I think doc_markdown should never lint any words inside quotes.

**Comment by mcarton** [comment]
This should work:
```rust
/// So much JSON:
/// ```json
/// { "many camelCase": "wow" }
/// ```
```

**Comment by mcarton** [comment]
Closing as IMO, this works as expected. Feel free to reopen and comment if you don't agree.
